File: C++/Product.cpp
class Product{

    /* Private Attribute */
    private:

        string price;
        string idProduct;

    /* Public Attribute and Method */
    public:

        /* Constructor */
        Product() {
            this->price = "Price";
            this->idProduct = "Id Product";
        }

        /* Setter dan Getter Price */
        void setPrice(string price) {
            this->price = price;
        }

        string getPrice() {
            return this->price;
        }

        /* Setter dan Getter Id Product */
        void setIdProduct(string idProduct) {
            this->idProduct = idProduct;
        }

        string getIdProduct() {
            return this->idProduct;
        }

        /* Destructor */
        ~Product() {

        }

};

File: PHP/Product.php
<?php

class Product {

    /* Private Attribute */
    private $price = "Price";
    private $idProduct = "Id Product";

    /* Constructor */
    public function __construct($price = "Price", $idProduct = "Id Product") {
        $this->price = $price;
        $this->idProduct = $idProduct;
    }

    /* Setter dan Getter Price */
    public function setPrice($price) {
        $this->price = $price;
    }

    public function getPrice() {
        return $this->price;
    }

    /* Setter dan Getter Id Product */
    public function setIdProduct($idProduct) {
        $this->idProduct = $idProduct;
    }

    public function getIdProduct() {
        return $this->idProduct;
    }

    /* Destructor */
    public function __destruct() {

	}

}

?>